src/components/common/Navbar/Navbar.tsx

```diff
@@ -22,12 +25,38 @@ const links = [
   },
 ]
 
+function NavLinks() {
+  return (
+    <nav className="navlinks__list">
+      <UIList>
+        {links.map((item) => (
+          <li key={item.name}>
+            <Link variant="display" key={item.href} href={item.href}>
+              {item.name}
+            </Link>
+          </li>
+        ))}
+      </UIList>
+    </nav>
+  )
+}
+
 function Navbar() {
   const [showMenu, setShowMenu] = useState(false)
+  const [isMobile, setIsMobile] = useState<boolean>(false)
+  const { width: screenWidth } = useWindowDimensions()
+  let onDismissTransition: () => unknown
+
+  useEffect(() => {
+    if (screenWidth) {
+      // notebook breakpoint = 1280px (See breakpoints on styles/global.scss)
+      setIsMobile(screenWidth < 1280)
```

Comment:
Here's something that worked for me in another [PR](https://github.com/vtex-sites/base.store/blob/684147555bd45e985690f721aca8799a6fafbb4f/src/components/sections/ProductGallery/ProductGallery.tsx#L34-L39):

```suggestion
      const notebookBreakpoint =
	      getComputedStyle(document.documentElement).getPropertyValue(
	        '--breakpoint-notebook'
	      ) || '1280'
      setIsMobile(screenWidth < parseInt(notebookBreakpoint, 10))
```

Reply:
Isn't better to remove this logic to a custom hook? Something like this...

```
const useIsMobile = () => {
	 const [isMobile, setIsMobile] = useState<boolean>(false)

  useEffect(() => {
    if (!screenWidth) {
      return
    }

    const notebookBreakpoint =
      getComputedStyle(document.documentElement).getPropertyValue(
        '--breakpoint-notebook'
      ) || '1280'

    setIsMobile(screenWidth < parseInt(notebookBreakpoint, 10))
  }, [screenWidth])

return isMobile
}
```

Reply:
I like this idea of the custom hook because I think we'll need it in more places.

Reply:
Me too! Love it! I will try that! Thanks guysss 🥳

Reply:
I decided to export a `isMobile` variable from `useWindowDimensions` hook :)  it'll be easier to apply wherever we need.